tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -9,47 +9,47 @@ import (
 	awssession "github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/client"
 	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
-	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/openshift"
 )
 
-// cloudProviderInfo is an provider specific struct which contains clients for EC2, IAM,
-// and the existing OpenShift cluster that is running on EC2.
+// awsProviderInfo is a provider specific struct which contains clients for EC2, IAM, and the existing OpenShift
+// cluster that is running on EC2.
 // This is an implementation of the Cloud interface.
-type cloudProviderInfo struct {
+type awsProviderInfo struct {
```

Comment:
Just call this `aws`

Reply:
It would conflict with `aws` package which is used very frequently here.

Reply:
Then name it `awsProvider`